main/templates/main/index.html

```diff
@@ -0,0 +1,430 @@
+                  title: '<strong style="color: white"><u>Dan Inon</u></strong>',
+                  width: 800,
+                  html: "<h2 style='color: white'>I'm a 2rd year Computer Science Student</h2>",
+                  showCloseButton: true,
+                  focusConfirm: false,
+                  confirmButtonText: '<i class="fa fa-thumbs-up""></i> Great!',
+                  confirmButtonAriaLabel: 'Thumbs up, great!',
+                })
+            }
+        </script>
+        
+	</head>
+	<body>
+        <nav role="navigation">
+          <div id="menuToggle">
+            <input type="checkbox" />
+            <span></span>
+            <span></span>
+            <span></span>
+            <ul id="menu">
+              <a href="#"><li><i class="fa fa-home" aria-hidden="true" style="font-size:24px;"></i>&nbsp;&nbsp; Home</li></a>
+              <a href="#"><li><i class="fa fa-id-card" aria-hidden="true" style="font-size:24px;"></i>&nbsp; Contact</li></a>
+              <a href="#"><li><i class="fa fa-user-plus" aria-hidden="true" style="font-size:24px;"></i>&nbsp; Register</li></a>
+                <a href="#"><li><i class="fa fa-sign-in" aria-hidden="true" style="font-size:24px;"></i>&nbsp;&nbsp; Log In</li></a>
+            </ul>
+          </div>
+        </nav>
+        
+		<div class="bgimg-1">
+		  <div class="header" id="nav">
+			  <a href="#index" class="logo" style="font-family: 'Kaushan Script', cursive;">
+                  <span style="font-weight: normal; font-family: 'Kaushan Script', cursive;">Lets</span> Meet
+              </a>
+			  <div class="header-right">
+				<a class="active" href="#home">Home</a>
+				<a href="#second">Contact</a>
+				<a href="#footer">About</a>
+                <a href="#log in">Log In</a>
+                <a href="#register">Register</a>
+			  </div>
+		  </div>
+		  <div class="caption">
+              <span class="text" style="font-family: 'Kaushan Script', cursive;">Lets Meet</span>
+		  </div>
+		</div>
+
+        <div class="bgimg-1 drop-shadow">
+            <div class="glass"></div>
+            <span id="glassText">
+              There is a difficulty coordinating meetings between different people, whether it’s a meeting between teammates, <br>
+                meeting between a few students, catching up with friends or family and more, due to a busy schedule <br>
+                that varies between person to person. <br>
+                Therefore, in order to solve this problem we would like to develop a platform named “Lets Meet”. <br>
+                The platform is meant to help the users to manage meetings of different types <br>
+                and centralize related information that links to those meetings such as files, chat and links to the meetings. <br>
+                Our site is meant to be used by everyone. 
```

Comment:
when I opened the website locally this line was hidden by the next div.
maybe it will be better to decrease the font size for all the text on this page.
please go over the rest of the page and make sure there are no other places with this issue

Reply:
change the layout, so the text is bounded, and the text size
Because of screen ration and size on my computer monitors there is no problem otherwise I would have noticed.
For now in smaller screen the text can be too big, I will fix this problem later.